docs/matchers/watson.md

```diff
@@ -0,0 +1,161 @@
+# IBM Watson Matcher
+
+[IBM Watson](https://www.ibm.com/watson) is an NLP API for matching strings to [intents](https://cloud.ibm.com/docs/services/assistant?topic=assistant-intents). Intents are created on your assistant intent page.
+
+## Configuring opsdroid
+
+In order to enable Watson skills, you must specify an `access-token`, an `assistant-id` and a `gateway` for your bot in the parsers section of the opsdroid configuration file.
+You can find these informations inside your Assistant Settings under the `API Details`. Note that depending where your bot is located the gateway will be different, just use this. For example: `gateway-fra`.
```

Comment:
You've got `gateway-fra` here and `gateway-fr` further down. Are these both valid?